core/conversion/conversionctx/ConversionCtx.cpp

```diff
@@ -124,13 +124,13 @@ ConversionCtx::ConversionCtx(BuilderSettings build_settings)
         settings.enabled_precisions.find(nvinfer1::DataType::kFLOAT) == settings.enabled_precisions.end(),
         "DLA supports only fp16 or int8 precision");
     cfg->setDLACore(settings.device.dla_core);
-    if (settings.dla_sram_size != 1048576){
+    if (settings.dla_sram_size != 1048576) {
```

Comment:
@narendasan : I see hard-coded values at multiple places. Can we move this to some macro definition instead?

Reply:
Sure thats fine, can you do this in a separate PR?

Reply:
Yeah, I will do that.